commit 5fee254098c11a52ce04e1efdc8c090caf592bfd
Author: Jason Cooper <jason@lakedaemon.net>
Date:   Tue Sep 14 09:45:51 2010 -0400

    staging: brcm80211: fix remaining checkpatch errors.
    
    Remaining errors are due to the use of typedefs.  They should dissappear
    once the typedefs get cleaned up.
    
    Signed-off-by: Jason Cooper <jason@lakedaemon.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/bcmutils.h
+++ b/drivers/staging/brcm80211/include/bcmutils.h
@@ -415,15 +415,15 @@ extern "C" {
 }
 
 #ifndef ABS
-#define	ABS(a)			(((a) < 0) ? -(a):(a))
+#define	ABS(a)			(((a) < 0) ? -(a) : (a))
 #endif				/* ABS */
 
 #ifndef MIN
-#define	MIN(a, b)		(((a) < (b)) ? (a):(b))
+#define	MIN(a, b)		(((a) < (b)) ? (a) : (b))
 #endif				/* MIN */
 
 #ifndef MAX
-#define	MAX(a, b)		(((a) > (b)) ? (a):(b))
+#define	MAX(a, b)		(((a) > (b)) ? (a) : (b))
 #endif				/* MAX */
 
 #define CEIL(x, y)		(((x) + ((y)-1)) / (y))
--- a/drivers/staging/brcm80211/sys/wl_mac80211.c
+++ b/drivers/staging/brcm80211/sys/wl_mac80211.c
@@ -1611,14 +1611,14 @@ static void __devexit wl_remove(struct p
 }
 
 static struct pci_driver wl_pci_driver = {
- name:	"brcm80211",
- probe: wl_pci_probe,
+ .name  = "brcm80211",
+ .probe = wl_pci_probe,
 #ifdef LINUXSTA_PS
- suspend: wl_suspend,
- resume:  wl_resume,
+ .suspend = wl_suspend,
+ .resume  = wl_resume,
 #endif				/* LINUXSTA_PS */
- remove:   __devexit_p(wl_remove),
- id_table: wl_id_table,
+ .remove   = __devexit_p(wl_remove),
+ .id_table = wl_id_table,
 };
 #endif				/* !BCMSDIO */
 
